plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java

```diff
@@ -147,6 +157,44 @@ public void close()
         stop();
     }
 
+    private static class HiveHadoopMinioState
```

Comment:
How about changing `HiveHadoop hiveMinioDataLake` in `BaseDeltaLakeConnectorSmokeTest` to `HiveHadoop` instead?

Reply:
good point 
we should stop using HiveMinioDataLake if we have to work against convenience this class is supposed to provide